packages/Icon/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+- Ellipsis vertical icon [@nahumzs](https://github.com/nahumzs).
```

Comment:
Also Character, Number, DateTime? Will you also follow up by removing them all from Wasabicons?

Reply:
I think we should keep the wasavicon icon we are the main users, but we are also allowing to overriding this, the component should work out of the box as all other ones.

Reply:
If an icon is in Paprika though, it will automatically get pulled into Wasabicons. So we can remove it from the wasabicons repo and just let it get imported from here, rather than have two copies that might get out of sync.

Reply:
ohh I see, I understand incorrectly what you mean, so you are saying that I have to go and remove these icons from wasabicons?

Reply:
Yes, exactly. No rush, just a good neighbourly thing to do 🙂

Reply:
Will add a ticket for it.